test/integration/destroy-test.js

```diff
@@ -60,3 +61,46 @@ test('destroy account', function (t) {
 
   .catch(t.error)
 })
+
+test('destroy account even when session is invalid', function (t) {
+  store.clear()
+  t.plan(5)
+
+  // mock the Date object to always return 1970-01-01T00:00:00.000Z
+  var clock = lolex.install(0)
+  var account = new Account({
+    url: baseURL,
+    id: 'abc4567'
+  })
+
+  nock(baseURL)
+  .delete('/session/account')
+  .reply(204)
```

Comment:
you can remove these lines, we don’t need to mock anything, because we don’t send any request to the server